package.yml

```diff
@@ -25,4 +25,11 @@ pages:
 
 requires:
     redaxo: '^5.0'
-    php: '>=5.5'
\ No newline at end of file
+    php: '>=5.5'
```

Comment:
Bei der PHP-Version scheint es üblicher zu sein, die Grenze doch nach oben offen zu lassen. Zumindest sehe ich das meist so in den composer.json's.
Ein Beispiel, Symfony: https://github.com/symfony/symfony/blob/master/composer.json#L19